app/assets/stylesheets/application.css.scss

```diff
@@ -240,6 +240,86 @@ ul.no-bullets {
   width: fit-content;
 }
 
+li.address {
+  margin: 0;
+  padding-left: 2.35em;
+  list-style: none;
+  background-image: url('icons/basic_geolocalize-05.svg');
+  background-repeat: no-repeat;
+  background-position: 0 0.2em;
+  background-size: 1.625em;
+}
```

Comment:
Refactoring the SVG icons to be list bullets.  This helped make the indentation line up with the first line of text when the text overflows onto a second line.